pkg/chain/celo/config.go

```diff
@@ -0,0 +1,37 @@
+package celo
+
+import (
+	"fmt"
+	"github.com/celo-org/celo-blockchain/common"
+	"github.com/keep-network/keep-common/pkg/chain/ethlike"
+)
+
+// Config is a struct that contains the configuration needed to connect to an
+// Celo node. This information will give access to a Celo network.
+type Config struct {
+	ethlike.CommonConfig
```

Comment:
That was my first approach but referencing the base struct from the client code looks weird then. In most cases, it will be `config.Config` which may be confusing. This is why I added the `Common` prefix.

Reply:
Where are the problematic references? I can't find them. Given that we are embedding, all fields should be copied and I think we shouldn't end up with `config.CommonConfig` or `config.Config` 🤔

Reply:
There are no such references for now but I tried to foresee potential inconveniences in the future. Anyway I changed as you proposed https://github.com/keep-network/keep-common/commit/b38a0217bdccaf07ff64240c88f2047473a00d21